src/GitHubUtil.js

```diff
@@ -219,6 +219,12 @@ function getReference(ref) {
                 reject(new ErrorContext(err, getReference.name, params));
                 return;
             }
+            // If the ref does not exist, but  but existing refs start with ref,
+            // an array is returned.
+            if (res.data === undefined || res.data.object === undefined) {
```

Comment:
If possible, please use some direct test like `res instanceof Array`.

Otherwise, replace `foo === undefined` with `!foo` to simplify and capture more unexpected cases.

Reply:
Done with Array.isArray() check.